### ISSUE TITLE
TFT results are not same 0.10.2 and 0.10.1 

### ISSUE THREAD

**Issue by sirison09** [jdb78/pytorch-forecasting#1036]
- PyTorch-Forecasting version: 0.10.2 and 0.10.1 
- 
- PyTorch_lightning  version: 1.6.4
- Python version: 3.8.12
- Operating System: Windows 

### Expected behavior

same results and same validation loss in fit functions for the versions of package 

### Actual behavior

Validation loss is not matching for the same code and same data

**Comment by babaniyi** [comment]
This is because it is using neural networks and you won't get the same "exact" results even if you run the same code in the same version of the package. The results will be approximately the same not exactly the same.

> 

This is because it is using neural networks and you won't get the same "exact" results even if you run the same code in the same version of the package. The results will be approximately the same not exactly the same.

**Comment by Metaming** [comment]
Hi sirison09, could you share how you set the random seed for the tft? Great thanks!
